src/Resources/views/Error/error_layout.html.twig

```diff
@@ -0,0 +1,83 @@
+{% spaceless %}
+<!DOCTYPE html>
+<html>
+<head>
+    {% block head %}
+    <meta charset="utf-8">
+    <title>{% block title %}{{ statusCode }} - ERROR{% endblock %}</title>
+    <meta name="generator" content="Contao Open Source CMS">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800,300' rel='stylesheet' type='text/css'>
+    <style>
+        {% include '@ContaoCore/error.css.twig' %}
```

Comment:
Umm... good question. Maybe because I added it later on and simply put it next to the reset CSS.

However we might want to add a `Css` folder etc. where we put the "flexible" stuff etc.